libs/auth-api-lib/src/lib/delegations/models/delegation.model.ts

```diff
@@ -52,6 +55,14 @@ export class Delegation extends Model {
   })
   toName!: string
 
+  @Column({
+    type: DataType.STRING,
+    allowNull: true,
```

Comment:
Should this be nullable? It's not in the migration.

Reply:
Fixed.